src/auth/auth.module.ts

```diff
@@ -13,8 +13,8 @@ import { EmployeesModule } from 'src/employees/employees.module';
     UserCredentialsModule,
     PassportModule,
     JwtModule.register({
-      secret: 'SECRET', // TODO: to put in env variables
-      signOptions: { expiresIn: '60S' }, // TODO: put in config
+      secret: `SeCR3TK3YF0RAp1@vhris`,
```

Comment:
@vision-jm 
kung walang downside na naka-expose sa repo ung secret key, baka mas ok na nasa env file nalang sya, what do you think?